### ISSUE TITLE
Node v0.12

### ISSUE THREAD

**Issue by T4d3o** [eendeego/node-openvg-canvas#25]
Hello. 
You plan to add support for node v0.12?

Thanks.

**Comment by schme16** [comment]
+1
I'd be pretty keen to get 0.12 support :)

**Comment by eendeego** [comment]
Yes, I plan to do that, specially migrating to https://github.com/rvagg/nan.

**Comment by itzaks** [comment]
+1 Any updates here? :)

**Comment by eendeego** [comment]
node-openvg has now been ported no _nan_, however, due to the way that TypedArrays are implemented now, I think I will have to port all JS code that depends on them to C++.

Until then, this is currently a blocker: https://github.com/nodejs/nan/issues/179
